### DISCUSSION TITLE
Errors from building from source

### DISCUSSION THREAD

**Discussion by hugolin615** [GMLC-TDC/HELICS#2385] [General]
Trying to install HELICS in Ubuntu 22.04 by building from source. 
There are no errors in the compiling and building. However, errors show up when I run the run3.sh:

./run3.sh: 5: ../../build/bin/comboFed: not found
./run3.sh: 7: ./run3.sh: 6: ../../build/bin/comboFed: not found
../../build/bin/comboFed: not found

It seems that an executable is not built. Any idea what the problem is.

Thank you

**Comment by phlptp** [comment]
Those shell scripts are part of the test suite.  They are meant to be run in the build directory, and as an instructive example.  You also would have to enable the examples to be built which is not part of the default build since they are not needed for anything
`-DHELICS_BUILD_EXAMPLES=ON`

I would also encourage checking out the examples repo https://github.com/GMLC-TDC/HELICS-Examples

and the user guide https://docs.helics.org/en/latest/user-guide/orientation.html
